specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/botservice.json

```diff
@@ -1012,6 +1012,283 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.BotService/checkEnterpriseChannelNameAvailability": {
+      "get": {
+        "tags": [
+          "Enterprise Channel"
+        ],
+        "description": "Check whether an Enterprise Channel name is available.",
+        "operationId": "EnterpriseChannels_GetCheckNameAvailability",
```

Comment:
Please update the name of this operation group to be singular instead of plural.

Reply:
Done

Reply:
Actually, when I try to change the name to EnterpriseChannel_GetCheckNameAvailability I get the following warning.

WARNING (OperationIdNounConflictingModelNames/R2063/SDKViolation): OperationId has a noun that conflicts with one of the model names in definitions section. The model name will be disambiguated to 'EnterpriseChannelModel'. Consider using the plural form of 'EnterpriseChannel' to avoid this. Note: If you have already shipped an SDK on top of this spec, fixing this warning may introduce a breaking change.
    - file:///C:/src/azure-rest-api-specs/specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/botservice.json:1022:8 ($.paths["/providers/Microsoft.BotService/checkEnterpriseChannelNameAvailability"].get.operationId)

What do you suggest?

Reply:
@NickEricson 
Regarding this operation - are you sure it should be a GET and not a POST?
According to this PR - maybe it should be POST:
https://github.com/Azure/azure-rest-api-specs/pull/3781

Additionally - "GetCheckNameAvailability" is a bit of a mouthful - I would change this to just "CheckNameAvailability"

Reply:
yes, good catch.